webcompat/static/css/src/issue-wizard-animations.css

```diff
@@ -29,66 +29,6 @@
   }
 }
 
-@keyframes slidedown {
-  0% {
-    opacity: 0;
-    z-index: -1;
-  }
-  99% {
-    z-index: -1;
-  }
-  100% {
-    margin-top: var(--between-steps-margin);
-    opacity: 1;
-    z-index: 0;
-  }
-}
-
-@keyframes slideup {
-  0% {
-    margin-top: var(--between-steps-margin);
-    z-index: -1;
-  }
-  100% {
-    margin-top: var(--presumed-high-step-height);
-    opacity: 0;
```

Comment:
`slidedown` and `slideup` keyframes are replaced with js animations and instead of margin-top, translateY is being modified (that helped to speed it up)